docs/writing_a_plugin.md

```diff
@@ -10,9 +31,9 @@ There are three possible RPC methods that Qlik can call.
 
 But what is the difference between a _script function_ and a _plugin function_?  
 
-For script functions you write any script directly in the Qlik client, and pass it as a parameter to the SSE plugin in one of the many pre-defined script functions. The plugin functions, on the other hand, are defined and implemented on the plugin side. The user calls the function directly by name and inserts the data as parameters in the Qlik client. The two options are described in detail below.
+For script functions you write any script directly in the Qlik client, and pass it as a parameter to the SSE plugin in one of the many pre-defined script functions. The plugin functions, on the other hand, are defined and implemented on the plugin side. The user calls the function directly by name and inserts the data as parameters in the Qlik client. The two options are described in details below.
```

Comment:
I think the previous text was correct (but don't trust my English too much). I would prefer "The two options are described in **detail** below" without the added "s".

Reply:
My mistake, I'll fix that :)